rules/eslint/best-practices/on.js

```diff
@@ -49,15 +49,15 @@ module.exports = {
     // disallow unnecessary function binding
     "no-extra-bind": "error",
     // disallow unnecessary labels
-    "no-extra-label": "off",
+    "no-extra-label": "error",
     // disallow fallthrough of case statements
     "no-fallthrough": "error",
     // disallow the use of leading or trailing decimal points in numeric literals
-    "no-floating-decimal": "off",
+    "no-floating-decimal": "error",
     // disallow assignments to native objects or read-only global variables
     "no-global-assign": "off",
     // disallow the type conversions with shorter notations
-    "no-implicit-coercion": "off",
+    "no-implicit-coercion": ["error", { "allow": [ "!!" ] }],
```

Comment:
I'm tempted to even get rid of `!!` but I think banning at least some types of implicit conversion makes code much more approachable.